rslib/src/scheduler/states/relearning.rs

```diff
@@ -134,7 +134,9 @@ impl RelearnState {
 
     fn answer_easy(self, ctx: &StateContext) -> ReviewState {
         let scheduled_days = if let Some(states) = &ctx.fsrs_next_states {
-            let (minimum, maximum) = ctx.min_and_max_review_intervals(1);
+            let (mut minimum, maximum) = ctx.min_and_max_review_intervals(1);
+            let good = ctx.with_review_fuzz(states.good.interval as f32, minimum, maximum);
+            minimum = good + 1;
```

Comment:
I have copied this code from learning.rs but I think that we need the following to deal with the case where `good` = `maximum`.

```rs
maximum = maximum.max(minimum);
```